docs/tutorials/dronecan_px4_flight_controller.rst

```diff
@@ -0,0 +1,449 @@
+which :ref:`ESC index <dronecan_px4_tutorial_esc_index>` corresponds to which motor on the airframe. Note that the ESC indices start from 0, while the ESC and motor numbers listed
+in QGroundControl start from 1. So ESC 1 in QGroundControl corresponds to the module with an ESC index of 0. In the ESC list on the actuators tab, you must set the function corresponding
+to each ESC index to the appropriate motor.
+
+For example, if you had a module that you wanted to be motor 1 on the airframe, following the convention established in the :ref:`ESC index <dronecan_px4_tutorial_esc_index>` section, 
+you should set its :ref:`ESC index <dronecan_px4_tutorial_esc_index>` to 0 in the IQ Control Center. Then on the actuators tab, you should set ESC 1 to the Motor 1 function.
+For the module you want to be motor 2, set its ESC index to be 1, and set ESC 2 to the Motor 2 function, and so on for the remaining modules.
+
+.. note:: Depdending on your modules :ref:`arming <manual_advanced_arming>` configurations, you may hear the module play its arming song when you
+    set the function for its ESC, if the module is connected to the bus. See the note in :ref:`dronecan_px4_fc_configuration` for more details on this.
+
+For this example, only 1 motor was used during testing, with an ESC index of 0. The image below shows how the actuator tab was set up to accomodate this
+
+.. figure:: ../_static/tutorial_images/dronecan_px4_tutorial/px4_dronecan_actuator_tab_esc_function_example.png
+    :align: center
+    :width: 60%
+    :alt: Example Actuator Tab ESC Function Setup
+
+    Example Actuator Tab ESC Function Setup for 1 Module With ESC Index 0
+
+If you prefer to interact directly with PX4 parameters, the functions can be assigned using the *UAVCAN_EC_FUNCX* parameters. An example of setting *UAVCAN_EC_FUNC1* is shown 
+in the image below.
+
+.. figure:: ../_static/tutorial_images/dronecan_px4_tutorial/px4_uavcan_ec_func1_parameter_qgc.png
+    :align: center
+    :width: 60%
+    :alt: UAVCAN_EC_FUNC1 Parameter
+
+    UAVCAN_EC_FUNC1 Parameter Setup for 1 Module With ESC Index 0
+
+The minimums and maximums set the range of values that will be sent in the :ref:`uavcan.equipment.esc.RawCommand <dronecan_messages_raw_command>` message to control the modules. Generally,
+the default minimum and maximum are fine and should not need to be changed.
+
+Other Configurations
+*********************
+There are various other configurations available under the UAVCAN section of Parameters, but none of them are essential for using DroneCAN with a Vertiq module. 
+Refer to the `PX4 parameter reference documentation <https://docs.px4.io/main/en/advanced_config/parameter_reference.html>`_ for more information on each of these parameters, 
+they may be useful in some applications.
+
+One other parameter of note is UAVCAN_NODE_ID. This sets the flight controllers' node ID. Generally, this can be left at its default of 1 as long as none of the 

```

Comment:
if this one is of note, I'd put this paragraph above the "there are various other..." one.